ChangeLog

```diff
@@ -0,0 +1,656 @@
+Version: 1.3-15 (2017-04-13, released)
```

Comment:
Delete this file too and any other unwanted files.

Reply:
Done